epi-cms-property.html

```diff
@@ -0,0 +1,31 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-ajax/iron-ajax.html">
+<!-- TODO add loading animation and error handing, e.g. just use ix-loading-ui-placeholder -->
+<dom-module id="epi-cms-property">
+    <template>
+        <iron-ajax url="/api/episerver/page/{{pageId}}/property/{{propertyName}}"
+                   auto handle-as="text" id="epiServerContentRequest"
+                   on-response="handleResponse">
+        </iron-ajax>
+    </template>
+    <script>
+        Polymer({
+            is: 'epi-cms-property',
+            properties: {
+                propertyName: {
+                    type: String
+                },
+                pageId: { /** Optional */
+                    type: String,
+                    value: window.CMSCurrentPageId
+                },
+                _htmlContent: {
+                    type: String
+                },
+            },
+            handleResponse: function (data, requestWebComponent) {
+                this.innerHTML = requestWebComponent.response; // We don't want Shadow DOM to allow natural styling from rules in parent shadow scope. Polymer doesn't make it easy - https://github.com/Polymer/polymer/issues/222
```

Comment:
Since [it's been raised|https://bitbucket.org/Interxion/customerportal/pull-requests/623/ixuat-4918-text-box-for-ecm-pages/diff#comment-30639352] we might as well support non HTML content.
Just check the `requestWebComponent.xhr.getResponseHeader("Content-Type")` and if it's not HTML, rather use the `text` property.